hypothesize/utilities.py

```diff
@@ -2261,32 +2262,69 @@ def linhat(x, con, est, *args):
 
     return psihat
 
-def mcpKadjp(p,proc="Holm"):
+def mcpKadjp(p, k=1, proc = 'Holm'):
+    '''
```

Comment:
These should be triple _double_ quotes. For example,

````
"""
some string
"""

Reply:
Changed as per your suggestion.